src/styles/index.scss

```diff
@@ -2,2 +1,231 @@
+.page {
   margin: 0;
+  box-sizing: border-box;
+  scroll-behavior: smooth;
+}
+
+:root {
+  --main-text-color: #00acdc;
+}
+
+.header {
+  margin: 0;
+  text-transform: uppercase;
+  box-sizing: border-box;
+  background-color: #fff;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  padding-inline: 50px;
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 14px;
+  display: flex;
+}
+
+
+.nav__list {
+  justify-content: center;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  display: flex;
+}
+
+.nav__link {
+  color: #060b35;
+  align-items: center;
+  height: 60px;
+  margin-left: 20px;
+  text-decoration: none;
+  display: flex;
+
+  &:hover {
+    color: var(--main-text-color);
+  }
+}
+
+.is-active {
+  color: var(--main-text-color);
+  height: 60px;
+  text-decoration: none;
+  display: flex;
+  position: relative;
+
+  &::after {
+    content: '';
+    background-color: var(--main-text-color);
+    border-radius: 8px;
+    width: 100%;
+    height: 4px;
+    display: block;
+    position: absolute;
+    bottom: 0;
+    left: 0;
+  }
+}
+
+.card {
+  width: 200px;
+  background: rgba(255, 255, 255, 1);
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+  transition-duration: 300ms;
+
+  &__img {
+    background-position: center;
+    background-size: cover;
+    margin: 33px 20px 0;
+  }
```

Comment:
The `background-position: center;` and `background-size: cover;` styles are not needed here. They are used to control the positioning and size of background images. However, you have not specified any background image for the `.card__img` class.